### PR TITLE
Cindy / Added rate methods, tests and none value checks

### PR DESCRIPTION
**Pull request by cindyariyo**
Added methods and tests for the traffic per second and none value checks in the methods.

### CODE REVIEW

File: tests/exporter/collectors/test_network.py [hunk 1]
```diff
@@ -45,20 +44,36 @@ def test_get_traffic_in(self):
...
         """
         @cindyariyo
         Test if a string representation of networking metrics is returned.
         """
         network_collector = NetworkCollector()
-        traffic_in = network_collector.get_traffic_in()
-        traffic_out = network_collector.get_traffic_out()
+        traffic_in = network_collector.get_rate_traffic_in()
+        traffic_out = network_collector.get_rate_traffic_out()
         self.assertEqual(network_collector.__str__(), f"Inbound Traffic: {traffic_in} Mb/s, Outbound Traffic: {traffic_out} Mb/s")
```
Comment (max-romagnoli) [inline tests/exporter/collectors/test_network.py:77]: @cindyariyo @ljdzed 
We need to refactor this and make it similar to the RAM one:
image

**Review by max-romagnoli — CHANGES_REQUESTED** [review]

File: tests/exporter/collectors/test_network.py (hunk 1 above)
Comment (ljdzed) [reply tests/exporter/collectors/test_network.py:77]: I will aim to do this tonight. Will also look into using a class variable to store the previous traffics both for inbound and outbound bytes.

Comment (max-romagnoli) [reply tests/exporter/collectors/test_network.py:77]: @ljdzed @cindyariyo 
Could either of you make the change to the test before this evening?

**Comment by max-romagnoli** [comment]
Additional changes:
* @cindyariyo you probably forgot to call the new gauges in `exporter.py`. I just added that in for you.
* In my view the rate methods using sleep are good enough for now. I have added an issue for next sprint to add the class variable and close this PR as it is: #109

**Review by max-romagnoli — APPROVED** [review]
# LGTM
Great work! Ready to merge.

Just a reminder of the NIT: #109

**Pull request merged**